packages/react/src/tooltip/Tooltip.tsx

```diff
@@ -16,6 +16,7 @@ type TooltipProps = BoxProps<
     content?: ReactNode;
     defaultOpen?: boolean;
     delayDuration?: ComponentPropsWithRef<typeof TooltipRoot>["delayDuration"];
+    disabled?: boolean;
```

Comment:
please add a separate changeset with this commit about how we've added the disabled prop to tooltip